examples/index.html

```diff
@@ -1,10 +1,10 @@
 <!doctype html>
-<iframe width="600px" height="800px" src="http://gohone.com/Quiz/5407769cfae50de35b93a1c2?debug=true&embed=true" data-hone frameborder="0" data-resize="true">You Dont Support Iframes</iframe>
-<script src="../build/embed.js" defer></script>
+<iframe width="600px" height="800px" src="https://gohone.com/Quiz/5407769cfae50de35b93a1c2?debug=true&embed=true" data-hone frameborder="0" data-resize="true">You Dont Support Iframes</iframe>
```

Comment:
Are we positive everything loads and renders perfectly over **http*s*://**?

I seem to recall some issues with fonts or something not being served correctly over ssl.  The wordpress site still having https-issues makes me think we need to double check our app too.

If you'd like, I'm happy to help out with this tomorrow.